lib/Backend/Func.cpp

```diff
@@ -301,8 +303,10 @@ Func::Codegen(JitArenaAllocator *alloc, JITTimeWorkItem * workItem,
     Js::ScriptContextProfiler *const codeGenProfiler, const bool isBackgroundJIT)
 {
     bool rejit;
+    int rejitCounter = 0;
     do
     {
+        Assert(rejitCounter < 25);
```

Comment:
>rejitCounter [](start = 15, length = 12)

I don't quite get this - are we saying that repeated rejits upto 25 are allowed?